### PR TITLE
force lazy import where possible

### PR DESCRIPTION
**Pull request by landam**
This PR fixes some issues related to lazy import.

See https://grass.osgeo.org/addons/grass7/logs/

### CODE REVIEW

File: grass7/raster/r.hypso/r.hypso.py [hunk 1]
```diff
@@ -126,6 +123,10 @@ def findint(kl,f):
     return z
 
 def plotImage(x,y,image,type,xlabel,ylabel,title):
+    import matplotlib #required by windows
+    matplotlib.use('wx') #required by windows
```
Comment (neteler) [inline grass7/raster/r.hypso/r.hypso.py:127]:
```suggestion
    matplotlib.use('wxAGG') #required by windows
```
...as in grass7/display/d.vect.colbp/d.vect.colbp.py (?)

File: grass7/raster/r.width.funct/r.width.funct.py [hunk 2]
```diff
@@ -113,6 +110,10 @@ def main():
     print('Done!')
 
 def plotImage(x,y,image,type,xlabel,ylabel,title):
+    import matplotlib #required by windows
+    matplotlib.use('WXAgg') #required by windows
```
Comment (neteler) [inline grass7/raster/r.width.funct/r.width.funct.py:114]:
```suggestion
    matplotlib.use('wxAGG') #required by windows
```

File: grass7/vector/v.faultdirections/v.faultdirections.py [hunk 3]
```diff
@@ -47,13 +47,14 @@
...
 import numpy as np
-import matplotlib #required by windows
-matplotlib.use('wx') #required by windows
-import matplotlib.pyplot as plt
 import grass.script as gscript
 
 
 def main():
+    import matplotlib #required by windows
+    matplotlib.use('wx') #required by windows
```
Comment (neteler) [inline grass7/vector/v.faultdirections/v.faultdirections.py:55]:
```suggestion
    matplotlib.use('wxAGG') #required by windows
```

File: grass7/raster/r.width.funct/r.width.funct.py (hunk 2 above)
Comment (landam) [reply grass7/raster/r.width.funct/r.width.funct.py:114]: Yes, seems to be correct, see https://matplotlib.org/3.2.1/tutorials/introductory/usage.html#the-builtin-backends

Comment (landam) [reply grass7/raster/r.width.funct/r.width.funct.py:114]: Anyway, "The names of builtin backends case-insensitive;"...

File: grass7/raster/r.richdem.flowaccumulation/r.richdem.flowaccumulation.py [hunk 4]
```diff
@@ -130,4 +129,5 @@ def main():
     accum.write(_output, overwrite=gscript.overwrite())
 
 if __name__ == "__main__":
+    options, flags = gscript.parser()
```
Comment (wenzeslaus) [inline grass7/raster/r.richdem.flowaccumulation/r.richdem.flowaccumulation.py:132]: Why this change? Plus it seems unrelated to the lazy import.

Comment (landam) [reply grass7/raster/r.richdem.flowaccumulation/r.richdem.flowaccumulation.py:132]: The module depends on `richdem` library. The compilation (`--html-description`) fails when if `parser()` is called in `main()` function because of missing library on build server (linux logs, windows binaries).
